client/cmd/dexc/ui/config.go

```diff
@@ -56,19 +58,23 @@ func setNet(net string) string {
 
 // Config is the configuration for the DEX client application.
 type Config struct {
-	DataDir    string `long:"dir" description:"Path to application directory"`
-	Config     string `long:"config" description:"Path to an INI configuration file. default:[home]/dexc_mainnet.conf"`
-	DBPath     string `long:"db" description:"Database filepath. Database will be created if it does not exist."`
-	CertsPath  string `long:"certs" description:"Path to a JSON-formatted file linking DEX URL keys to TLS certificate filepaths."`
-	RPCOn      bool   `long:"rpc" description:"turn on the rpc server"`
-	RPCAddr    string `long:"rpcaddr" description:"RPCServer listen address"`
-	WebOn      bool   `long:"web" description:"turn on the web server"`
-	WebAddr    string `long:"webaddr" description:"HTTP server address"`
-	NoTUI      bool   `long:"notui" description:"disable the terminal-based user interface. must be used with --rpc or --web"`
-	Testnet    bool   `long:"testnet" description:"use testnet"`
-	Simnet     bool   `long:"simnet" description:"use simnet"`
-	ReloadHTML bool   `long:"reload-html" description:"Reload the webserver's page template with every request. For development purposes."`
-	DebugLevel string `long:"log" description:"Logging level {trace, debug, info, warn, error, critical}"`
+	DataDir     string `long:"dir" description:"Path to application directory"`
+	Config      string `long:"config" description:"Path to an INI configuration file. default:[home]/dexc_mainnet.conf"`
+	DBPath      string `long:"db" description:"Database filepath. Database will be created if it does not exist."`
+	CertsPath   string `long:"certs" description:"Path to a JSON-formatted file linking DEX URL keys to TLS certificate filepaths."`
+	RPCOn       bool   `long:"rpc" description:"turn on the rpc server"`
+	RPCAddr     string `long:"rpcaddr" description:"RPCServer listen address"`
+	RPCUser     string `long:"rpcuser" description:"RPCServer user name"`
+	RPCPassword string `long:"rpcpassword" description:"RPCServer password"`
+	RPCCert     string `long:"rpccert" description:"RPCServer certificate file location"`
+	RPCKey      string `long:"rpckey" description:"RPCServer key file location"`
```

Comment:
I didn't bring this up previously, but "RPCServer" isn't valuable to the user.  "RPC server" would be better since the user doesn't care about the internal type names.